sdks/builds/android.mk

```diff
@@ -81,6 +81,8 @@ _android_$(1)_CONFIGURE_FLAGS= \
 	$$(NDK_DIR)/build/tools/make-standalone-toolchain.sh  --platform=$(2) --arch=$(1) --install-dir=$$(TOP)/sdks/builds/toolchains/android-$(1) --toolchain=$(4)
 	touch $$@
 
+android-toolchain:: .stamp-android-$(1)-toolchain
```

Comment:
You can use single `:` as you don't have multiple commands, just multiple dependencies.

Reply:
I'll fix that in the follow up PR